### ISSUE TITLE
Slower performance on slides with pictures

### ISSUE THREAD

**Issue by mdh34** [Philip-Scott/Spice-up#203]
Great app! I've noticed than when a slide has one or more pictures, there's a noticeable delay in switching between it using the editor and presentation modes. I have a similar problem when dragging/resizing images but I don't know if that's just because of the performance of my computer.

**Comment by Philip-Scott** [comment]
Yeah, that's sadly the normal for now. Gtk loads the image from disk on each change..... But things will change once the new Canvas tech is done :) 

For now, if you use smaller images things will not be as laggy!
